### ISSUE TITLE
device error

### ISSUE THREAD

**Issue by jiyu3959** [HarryK24/CW-pytorch#3]
error: indices should be either on cpu or on the same device as the indexed tensor (cpu)

one_hot_labels = torch.eye(len(outputs[0]))[labels].to(device))

I don't know, which item should set to cpu

**Comment by youyuanyi** [comment]
change the code to this:
w = torch.tensor(torch.zeros_like(input_image, requires_grad=True)).to(self.device)
